services/TransmittalService.js

```diff
@@ -16,27 +16,35 @@ module.exports = {
                     day = timestamp.substring(6,8),
                     hour = timestamp.substring(8,10),
                     minute = timestamp.substring(10,12);
-           
+
                 var timestampDate = new Date(year, month - 1, day - 1, hour, minute);
-          
+
                 if (timestampDate > data.timestamp) {
                     result.result = true;
                 }
-            } 
-            
+            }
+
             return callback(null, result);
         });
     },
     isTaxIDTheSameAsLastYear: function(activityYear, respondentID, taxID, callback) {
-        Transmittal.findOne({'activity_year': activityYear - 1, 'respondent_id': respondentID, 'tax_id': taxID}, function(err, data) {
+        Transmittal.findOne({'activity_year': activityYear - 1, 'respondent_id': respondentID}, function(err, data) {
             if (err) {
                 return callback(err, null);
             }
-
+            var result = {
+                'respondentID': respondentID,
+                'taxID': taxID,
+                'Previous Year Tax ID': '',
```

Comment:
I know that since we don't have data for previous years in the db this has to return an empty string, but shouldn't there be a `//TODO` or something here to note why we're returning the empty string now?